### DISCUSSION TITLE
Package questions (x3270, xtrs)

### DISCUSSION THREAD

**Discussion by n3gwg** [Homebrew/discussions#3214] [Everyday usage]
Brew maintainers/developers:

First, let me start off by saying thank you to all the developers that work so hard to make brew as usable and excellent as it is. Now, my questions! :)

1. I am curious what level of effort might be requisite in pursuance of seeing the addition of xtrs to brew?

2. A long time ago I had asked if the implementation of x3270 that is inclusive to brew could have the GUI version of added and was told something along the lines "it's too hard, go away". At the time I was rather surprised to see such a response and just figured ok and installed it a different way. Thus, I will ask again, presuming maybe people feel more capable or new and better developers and maintainers are involved, can the graphical version of x3270 be added to brew?

Thanks in advance to anyone that addresses either of these concerns.

Stuart

**Comment by SMillerDev** [comment]
For 1 we have some documentation: https://docs.brew.sh/Adding-Software-to-Homebrew

For issue 2, we don't generally add GUIs to programs in brew. If there is a upstream installer for the GUI version you can use the docs [here](https://docs.brew.sh/Adding-Software-to-Homebrew#casks) though
